### DISCUSSION TITLE
Is it possible to have a second reference to a footnote?

### DISCUSSION THREAD

**Discussion by martinscripts** [quarto-dev/quarto-cli#8408] [Q&A] [answered]
### Description

I would like to reference one footnote twice. 

```qmd
This is my text[^myfootnote].
It continues here.
Let's reference the footnote again[^myfootnote].

[^myfootnote]: This is my footnote.
```

If I just use the same key `[^myfootnote]` again, it will print the same footnote twice with two different numbers. 
Any ideas how to prevent this and instead have two references to the same note?

**Comment by mcanouil** [comment, marked as answer]
Which format?

Note that it is very likely something not up to Quarto (Pandoc behaviour discussed already in other threads).

**Reply by martinscripts** [reply]
Hi mcanouil, thanks for the fast answer.

I was trying in html format. 
Okay, I think I didn't quite understand the way quarto and pandoc work together, yet. 
I found it [here](https://github.com/jgm/pandoc/issues/224) now, possibly it's what you referred to.
Thanks, BR,
.Martin

**Reply by cderv** [reply]
@martinscripts This is a known limitation with Pandoc indeed. Tracked here in Quarto 
* https://github.com/quarto-dev/quarto-cli/issues/5440

**Reply by mcanouil** [reply]
Yes, and here in Quarto repository also, for example:
- https://github.com/quarto-dev/quarto-cli/issues/5440

**Reply by martinscripts** [reply]
whyever i didn't find that in my search. 🙈 Thanks!

**Comment by N1N74** [comment]
Hola, I just learnt that [github.com/markdown-it](https://github.com/markdown-it/markdown-it), is able to add duplicate footnotes and it works intuitively. It also distinguishes reused references and adds backlinks to them which you can follow from my suggestion here #14007. You can preview the [live demo on github.io](https://markdown-it.github.io/). Might be helpful if you really need it for something.

<img width="1738" height="466" alt="Image" src="https://github.com/user-attachments/assets/5b4deafd-5e56-4d81-854e-fafc27744f42" />
